pkg/kube/inject/testdata/inject/traffic-annotations.yaml

```diff
@@ -15,7 +15,7 @@ spec:
         traffic.sidecar.istio.io/excludeOutboundPorts: "7,8,9"
         traffic.sidecar.istio.io/includeOutboundIPRanges: "127.0.0.1/24,10.96.0.1/24"
         traffic.sidecar.istio.io/excludeOutboundIPRanges: "10.96.0.2/24,10.96.0.3/24"
-        istio.io/proxyConfig: |-
+        proxy.istio.io/config: |-
```

Comment:
Are we changing the annotation?

Reply:
Yes, it was not yet shipped. The new annotation is TOC approved

Reply:
ok perfect ... thanks!